frontend/src/pages/NicknamePage/index.tsx

```diff
@@ -1,13 +1,60 @@
-import React, { useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { useHistory } from "react-router-dom";
+import createPersistedState from "use-persisted-state";
 import TextField from "../../components/TextField";
 import Button from "../../components/Button";
 import styles from "./style.module.css";
+import { BrowserHistoryContext } from "../../App";
+import validateGame from "../../api/validateGame";
+import createPlayer from "../../api/createPlayer";
 
-const NicknamePage = () => {
-  const memoryHistory = useHistory();
+const useUserIdState = createPersistedState("userId");
+
+type Props = {
+  gameCode: string;
+};
 
+const NicknamePage = ({ gameCode }: Props) => {
   const [nickname, setNickname] = useState("");
+  const [error, setError] = useState("");
+  const [userId, setUserId] = useUserIdState("");
```

Comment:
I'm running into an error thrown from here testing locally - seems to be an issue with persisted state

Reply:
This appears to be resolved now! 🎉